### ISSUE TITLE
Figure out how we'd host ledgersync page

### ISSUE THREAD

**Issue by mattgmarcus** [LedgerSync/ledger_sync#91]

**Comment by ryanwjackson** [comment]
https://gohugo.io/hosting-and-deployment/hosting-on-github/

**Comment by ryanwjackson** [comment]
It looks like we can use hugo to build our website.  We will create a `docs` folder (or something similar) in our root and then we can deploy: https://gohugo.io/hosting-and-deployment/hosting-on-github/#deployment-of-project-pages-from-docs-folder-on-master-branch

We can also use customer domains using CNAMES: https://gohugo.io/hosting-and-deployment/hosting-on-github/#deployment-of-project-pages-from-docs-folder-on-master-branch

A few things:
- Have the files in this repo or another one.  Downside to another one is you don't get the nice github URL.
- I am not sure how bundling the gem together works, but I assume we would want to confirm that the directories for the site are not included in the gem (so it doesn't increase the size).  Although they may be helpful as they do contain docs and examples.

Basically, we have options and we just need a single, static page HTML/JS/CSS site. Hugo isn't our only option, but it's fair to assume our input to github or custom domains will be a static webpage.

@mattgmarcus Thoughts?

**Comment by mattgmarcus** [comment]
This seems good to me. If we can keep it all here without increasing the gem size, it sounds good

**Comment by mattgmarcus** [comment]
Let's use this theme: https://pages-themes.github.io/cayman/
https://github.com/pages-themes/cayman


Some other helpful links:
Adding a theme (I already did this) - https://help.github.com/en/github/working-with-github-pages/adding-a-theme-to-your-github-pages-site-using-jekyll
Managing a custom domain - https://help.github.com/en/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site

Let's first get it to where we have the page hosted on `ledgersync.github.io`, with the templates so that we can edit both the html and css

**Comment by ryanwjackson** [comment]
@mattgmarcus working on this now.

Do you want to set up a google analytics code for it?
